src/main/java/net/fabricmc/loom/task/RemapJarTask.java

```diff
@@ -80,9 +80,13 @@
 import net.fabricmc.tinyremapper.TinyRemapper;
 
 public abstract class RemapJarTask extends AbstractRemapJarTask {
+	@Deprecated
 	@InputFiles
 	public abstract ConfigurableFileCollection getNestedJars();
```

Comment:
We cannot deprecate this, as its used for adding files that arent part of a `Configuration`

Reply:
Adding to this collection directly doesn't add any metadata even if the file isn't actually a mod jar. Files can also be added directly to `Configuration`s. So it doesn't really make sense to me to actually keep this.